spec/rails_helper.rb

```diff
@@ -1,8 +1,8 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'simplecov'
 SimpleCov.start
-require 'login_helper'
-require 'phone_helper'
+require 'support/login_helper'
+require 'support/phone_helper'
```

Comment:
Do not need  require `support/login_helper` and `support/phone_helper`. The following line requires all files on support dir:
```
 Dir[Rails.root.join('spec', 'support', '**', '*.rb')].sort.each { |f| require f } 
```